src/Queries/Math/Add.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I suggest doing something similar to the `ConstantQuery` for all the `BinaryOp` classes. Instead of having one class for each type of `BinaryOp`, we can add a `$termType` argument to the `BinaryOp` constructor and have the places that construct those classes call `new BinaryOp` with the right type.